tests/README.md

```diff
@@ -49,7 +47,7 @@ tmt run
 This will use latest hirte packages from
 [hirte-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/) repository.
 
-## Running integration tests with local hirte build
+### Running integration tests with local hirte build
```

Comment:
This section shouldn't be a subsection of `Running integration tests` chapter, right? I think should be on the same level

Reply:
Changed.

Reply:
Haven't you forgot to push that change?

Reply:
Good catch! Was probably caused during rebase. Should be solved now.